packages/core/kind/debugClassnames.js

```diff
@@ -0,0 +1,49 @@
+/* !
+  classnames Copyright (c) 2018 Jed Watson.
+  Licensed under the MIT License (MIT), see
+  http://jedwatson.github.io/classnames
+*/
+
+const hasProp = {}.hasOwnProperty;
```

Comment:
Will want to change this to `Object.prototype.hasOwnProperty` to avoid upcoming ESLint `no-prototype-builtins` rule issue